scripts/carthage-archive.sh

```diff
@@ -24,7 +24,9 @@ if [ ! "$(printf '%s\n' "$REQUIRED_SWIFT_TOOLING" "$TOOLS_VERSION" | sort -V | h
 fi
 
 swift package generate-xcodeproj
```

Comment:
I think it could be better to use something like 
```suggestion
swift package generate-xcodeproj --xcconfig-overrides Config.xcconfig
```

And having the configurations inside it, did you consider that/try that?

Reply:
I like it! Didn't know SPM supports it. I'll give it a try

Reply:
I tried using custom Config.xcconfig and have some issues. Initially I tried using something like that:
```
PRODUCT_BUNDLE_IDENTIFIER = com.CombineCommunity.$(PRODUCT_NAME)
CURRENT_PROJECT_VERSION = 1
```

The `CURRENT_PROJECT_VERSION` seems to work just fine, but `PRODUCT_BUNDLE_IDENTIFIER` has no effect. 
And it seems that, autogenerated xcodeproj is using my config on all targets (including dependencies), and I don’t see a way to control that other that manually inspecting the project.

I also tried defining some custom settings, like `BUNDLE_IDENTIFIER_PREFIX`, and then using them in Info.plist, but then I discovered that generated project is using a different one, stored in Project.xcodeproj/TargetName_Info.plist. `INFOPLIST_PATH` also has no effect.